OpenRA.Mods.Common/Traits/Player/ProductionQueue.cs

```diff
@@ -508,7 +508,7 @@ public void Tick(PlayerResources pr)
 			if (pm.PowerState != PowerState.Normal)
 			{
 				if (--Slowdown <= 0)
-					Slowdown = Queue.Info.LowPowerSlowdown;
+					Slowdown = Queue.Info.LowPowerModifier;
```

Comment:
This means that the slowdown is now multiplied by 100. Every tick, the Slowdown will go down by 1, so it will now take 300 ticks to do one build tick on low power. Allowing modifiers that aren't effectively multiple of 100 will require careful work to allow for fractional build ticks. I think it may be possible to do by replacing `--Slowdown` with `Slowdown -= 100`, but I'm not 100% sure.

You asked on IRC about the interaction with #14702; I have nothing in principle against allowing a modifier to speed up the build queue (values less than 100). That would require the logic here to allow the build to tick multiple times per game tick.